### PR TITLE
#159082378 Users should be able to create their profiles

### PR DESCRIPTION
**Pull request by lytes20**
#### What does this PR do?
Creates a profile for user when a user registers
#### Description of Task to be completed?
Once a user is registered, their profile is created
The profile displays the bio, image/avatar, and username.
A user can view their profiles, edit it but can NOT edit another user's profile.
#### How should this be manually tested?
By creating a user and then retrieving their profile at this endpoint `profiles/<username>/`
#### Any background context you want to provide?
profiles are very helpful for displaying user information
#### What are the relevant pivotal tracker stories?
[#159082378](https://www.pivotaltracker.com/story/show/159082378)
#### Screenshots
screen shot 2018-08-09 at 11 14 36

### CODE REVIEW

File: authors/apps/authentication/serializers.py [hunk 1]
```diff
@@ -190,7 +195,6 @@ class Meta:
         # password field, we needed to specify the `min_length` and
         # `max_length` properties too, but that isn't the case for the token
         # field.
-        read_only_fields = ('token',)
```
Comment (geofrocker) [inline authors/apps/authentication/serializers.py:193]: why did you remove this?

File: authors/apps/authentication/serializers.py [hunk 2]
```diff
@@ -179,9 +179,14 @@ class UserSerializer(serializers.ModelSerializer):
...
     )
 
+    profile = ProfileSerializer(write_only=True)
+
+    bio = serializers.CharField(source='profile.bio', read_only=True)
+    image = serializers.CharField(source='profile.image', read_only=True)
+
     class Meta:
         model = User
-        fields = ('email', 'username', 'password', 'token',)
```
Comment (geofrocker) [inline authors/apps/authentication/serializers.py:184]: why did you remove this?

File: authors/apps/authentication/models.py [hunk 3]
```diff
@@ -82,12 +83,6 @@ class User(AbstractBaseUser, PermissionsMixin):
     # true
     is_verified = models.BooleanField(default=False)
 
-    # A timestamp representing when this object was created.
-    created_at = models.DateTimeField(auto_now_add=True)
-
```
Comment (geofrocker) [inline authors/apps/authentication/models.py:87]: why did you remove these fields

File: authors/apps/authentication/serializers.py (hunk 1 above)
Comment (lytes20) [reply authors/apps/authentication/serializers.py:193]: This was no longer needed code, it  use was that it was necessary when displaying the token along with user information

File: authors/apps/authentication/serializers.py (hunk 2 above)
Comment (lytes20) [reply authors/apps/authentication/serializers.py:184]: Same thing here, I didn't delete the entire line but rather deleted the token field and modified it to add other user profile fields. these fields are later displayed as user's information/profile.

File: authors/apps/authentication/models.py (hunk 3 above)
Comment (lytes20) [reply authors/apps/authentication/models.py:87]: The `created_at` and `updated_at` fields are shared by both the `User` model and `Profile` model. so i deleted them from the user model. Added them to a model file that can be shared by both apps as a user and a profile are created and updated at same times.

**Review by abulojoshua1 — APPROVED** [review]

**Review by ssewilliam — APPROVED** [review]
This looks good

File: authors/apps/authentication/models.py (hunk 3 above)
Comment (geofrocker) [reply authors/apps/authentication/models.py:87]: great
